tensorflow_addons/optimizers/moving_average_test.py

```diff
@@ -107,25 +107,30 @@ def test_model_weights_update(self):
         self.evaluate(mean_update)
         self.assertAllClose(model.variables[0].read_value(), [[0.9]])
 
+    def test_optimizer_string(self):
+        _ = MovingAverage('adam')
+
     def test_config(self):
         sgd_opt = tf.keras.optimizers.SGD(
             lr=2.0, nesterov=True, momentum=0.3, decay=0.1)
         opt = MovingAverage(
             sgd_opt,
             average_decay=0.5,
-            num_updates=100,
+            num_updates=None,
```

Comment:
Any reason why we change this from 100 to `None` for testing?

Reply:
`num_updates` isn't registered as a hyperparameter using `self._set_hyper()` specifically, because `num_updates` could take the value of `None` which was not compatible with `self._set_hyper()` which only supports numeric values IIRC. Wanted to make sure that passing a `None` to the `config` wouldn't create an issue.